### DISCUSSION TITLE
Add support for Misskey

### DISCUSSION THREAD

**Discussion by YokaiRick** [Traewelling/traewelling#2019] [Ideas / Feature Requests]
Maybe it is possible to Support Misskey. Misskey is a Fediverse Server similar to Mastodon.
Sadly it doesn't use Masto-Api, so login/connecting traewelling via Mastodon instance settings in the profile doesn't work. 

I know this is probably a big (maybe too big) feature request, but still wanted to propose it.

Some more Infos about Misskey:
GithupRepo: https://github.com/misskey-dev/misskey
API-Docs: https://misskey.io/api-doc

**Comment by ChristianTacke** [comment]
On the misskey github page it talks about activitypub. I think, that's the same protocol as mastodon uses for federation?

That said, maybe träwelling could start by supporting WebFinger?
* Maybe start with simple webfinger and the [profile-page](https://webfinger.net/rel/profile-page/)?
* And then look further at the [Mastodon WebFinger docs](https://docs.joinmastodon.org/spec/webfinger/)

**Comment by tibequadorian** [comment]
I think, there's a misunderstanding here. @YokaiRick talks about the login, which relies on the Mastodon API (clientside) and I think @ChristianTacke talks about federating over ActivityPub (serverside). These should be two different discussions.

I found this because I was also interested in federation over ActivityPub.
